vscode-dotnet-runtime-library/src/Acquisition/VersionResolver.ts

```diff
@@ -23,6 +30,60 @@ export class VersionResolver implements IVersionResolver {
         this.webWorker = new WebRequestWorker(extensionState, eventStream, this.releasesUrl, this.releasesKey);
     }
 
+    /**
+     * @remarks
+     * Use the release.json manifest that contains the newest version of the SDK and Runtime for each major.minor of .NET to get the available versions.
+     * Relies on the context listRuntimes to tell if it should get runtime or sdk versions.
+     *
+     * @params
+     * webWorker - This class can use its own web-worker or a custom one for testing purposes.
+     *
+     * @returns
+     * IDotnetListVersionsResult of versions available.
+     *
+     * @throws
+     * Exception if the API service for releases-index.json is unavailable.
+     */
+    public async GetAvailableDotnetVersions(commandContext: IDotnetListVersionsContext | undefined, webWorker: WebRequestWorker | undefined) : Promise<IDotnetListVersionsResult>

```

Comment:
Can we add unit test coverage for this method? It looks like there's an existing VersionResolver unit test class that can be expanded.

Reply:
Definitely! I added a pretty basic one, but I think it's an improvement because it tests the online functionality of it. The SDK test is a more comprehensive test.